### ISSUE TITLE
If no sources but binaries are available, consider scanning at least e.g. header files

### ISSUE THREAD

**Issue by sschuberth** [oss-review-toolkit/ort#3112]
I gave this some thought again, too. While I still do believe it makes sense to scan header files included in binary artifacts, we will basically have the same problem e.g. in binary Conan packages. So the more generic solution would probably be to continue to only add binary artifacts as binary artifacts in ORT, and not additionally as source artifacts, but add to ORT the (optional) feature to also scan (some files from) binary artifacts. This would at least make sense in the case where only a binary artifact is available, so we do scan at least the header files.

To summarize, I'd now vote to only add the binary artifact here, and implement the feature to scan header files from binary packages at another place in the scanner / downloader, so the logic applies to all package managers.

_Originally posted by @sschuberth in https://github.com/oss-review-toolkit/ort/pull/3099#r493255505_

**Comment by sschuberth** [comment]
Closing as not planned.
